cmd/mattermost/commands/plugin.go

```diff
@@ -0,0 +1,181 @@
+	RunE:    pluginEnableCmdF,
+}
+
+var PluginDisableCmd = &cobra.Command{
+	Use:     "disable",
+	Short:   "Disable a plugin",
+	Long:    "Disable a plugin",
+	Example: `  plugin disable hovercardexample`,
+	RunE:    pluginDisableCmdF,
+}
+
+var PluginListCmd = &cobra.Command{
+	Use:     "list",
+	Short:   "List plugins",
+	Long:    "List all active and inactive plugins",
+	Example: `  plugin list`,
+	RunE:    pluginListCmdF,
+}
+
+func init() {
+	PluginCmd.AddCommand(
+		PluginCreateCmd,
+		PluginDeleteCmd,
+		PluginEnableCmd,
+		PluginDisableCmd,
+		PluginListCmd,
+	)
+	RootCmd.AddCommand(PluginCmd)
+}
+
+func pluginCreateCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	if len(args) < 1 {
+		return errors.New("Expected at least one argument. See help text for details.")
+	}
+
+	for i, plugin := range args {
+		fileReader, err := os.Open(plugin)
+		if err != nil {
+			return err
+		}
+
+		if _, err := a.InstallPlugin(fileReader); err != nil {
+			return errors.New("Unable to create plugin: " + args[i] + ". Error: " + err.Error())
+		}
+		fileReader.Close()
+	}
+
+	CommandPrettyPrintln("Created plugin(s)")
+
+	return nil
+}
+
+func pluginDeleteCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	if len(args) < 1 {
+		return errors.New("Expected at least one argument. See help text for details.")
+	}
+
+	if err := a.RemovePlugin(args[0]); err != nil {
+		return errors.New("Unable to delete plugin: " + args[0] + ". Error: " + err.Error())
+	}
+
+	CommandPrettyPrintln("Deleted plugin")
+
+	return nil
+}
+
+func pluginEnableCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	if len(args) < 1 {
+		return errors.New("Expected at least one argument. See help text for details.")
+	}
+
+	for i, plugin := range args {
+		if err := a.EnablePlugin(plugin); err != nil {
+			return errors.New("Unable to enable plugin: " + args[i] + ". Error: " + err.Error())
+		}
+	}
+
+	CommandPrettyPrintln("Enabled plugin(s)")
+
+	return nil
+}
+
+func pluginDisableCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	if len(args) < 1 {
+		return errors.New("Expected at least one argument. See help text for details.")
+	}
+
+	if err := a.DisablePlugin(args[0]); err != nil {
+		return errors.New("Unable to disable plugin: " + args[0] + ". Error: " + err.Error())
+	}
+
+	CommandPrettyPrintln("Disabled plugin")
+
+	return nil
+}
+
+func pluginListCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	pluginsResp, _ := a.GetPlugins()
```

Comment:
Here, If we handle the error we can see more information. Is not the same that you have not active or inactive plugins, than the information that you can't list it because the  plugins are disable. I would prefer to use the same approach handling errors used in the other commands.